src/wallet/rpcwallet.cpp

```diff
@@ -2164,7 +2164,7 @@ UniValue settxfee(const UniValue& params, bool fHelp)
     if (fHelp || params.size() < 1 || params.size() > 1)
         throw runtime_error(
             "settxfee amount\n"
-            "\nSet the transaction fee per kB.\n"
+            "\nSet the transaction fee per kB. (Overwrites the paytxfee parameter)\n"
```

Comment:
Can you elaborate? The trivial commit shouldn't remove anything. It's a cherry-pick from my `cleanup-txfee-params` branch.